engine/classes/ElggRelationship.php

```diff
@@ -131,7 +131,7 @@ public function getURL() {
 			$subtype = $this->getSubtype();
 
 			$function = "";
-			$handlers = _elgg_config()->relationship_url_handler;
+			$handlers = _elgg_services()->config->relationship_url_handler;
```

Comment:
I believe this section can be removed from `ElggRelationship`

Reply:
think so too